docs/server/installation.rst

```diff
@@ -212,6 +212,10 @@ an alternative path as an argument if required.
 .. warning:: This default configuration is enough to experiment with Pootle.
    **Don't use this configuration in a production environment**.
 
+You can specify the parameters to set up your database if you don't want to use
```

Comment:
Hmmm reading this I have the feeling that a SQLite setup is a blessed option. Can we maybe reword it to omit SQLite at all?